diesel/src/macros/mod.rs

```diff
@@ -177,6 +177,23 @@ macro_rules! __diesel_column {
 /// # }
 /// ```
 ///
+/// If you are using types that aren't from Diesel's core types, you can specify
+/// which types to import. Note that the path given has to be an absolute path
+/// relative to the crate root. You cannot use `self` or `super`.
+///
+/// ```ignore
+/// #[macro_use] extern crate diesel;
+/// extern crate diesel_full_text_search;
+///
+/// table! {
+///     posts using types from diesel::types, diesel_full_text_search::types {
```

Comment:
I'm open to suggestions. When it was just table name and optionally the primary key, the macro rules themselves were pretty self documenting. Agreed it's obscure now. Do you think there's another syntax which is more obvious? Or are you just saying we need to document the full syntax better?